### PR TITLE
Update/readme.md

### PR DESCRIPTION
**Pull request by daiyanze**
Fixes #[issue number].

Changes proposed:
- update ./README.md for prompting `Modal.setAppElement` implementation
- update ./docs/accessibility/README.md to explain usage of `Modal.setAppElement`

Upgrade Path (for changed or removed APIs):

Acceptance Checklist:
- [ ] All commits have been squashed to one.
- [ ] The commit message follows the guidelines in `CONTRIBUTING.md`.
- [ ] Documentation (README.md) and examples have been updated as needed.
- [ ] If this is a code change, a spec testing the functionality has been added.
- [ ] If the commit message has [changed] or [removed], there is an upgrade path above.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -50,6 +50,9 @@ const customStyles = {
   }
 };
 
+// Make sure to bind modal to your appElement (http://reactcommunity.org/react-modal/accessibility/)
```
Comment (diasbruno) [inline README.md]: Perhaps this can be rephrased like: `In case you want to use aria-hidden, make sure...`.

Comment (daiyanze) [reply README.md]: Well, I think if without setting appElement, it will throw warning all the time. So I guess using aria-hidden should be a "must".

Comment (diasbruno) [reply README.md]: Hm, ok.

**Comment by diasbruno** [comment]
Thank you, @daiyanze!

**Pull request merged**